spark/v3.3/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSparkSqlExtensionsParser.scala

```diff
@@ -206,8 +206,9 @@ class IcebergSparkSqlExtensionsParser(delegate: ParserInterface) extends ParserI
             normalized.contains("write unordered") ||
             normalized.contains("set identifier fields") ||
             normalized.contains("drop identifier fields") ||
-            normalized.contains("create branch"))) ||
-            normalized.contains("replace branch")
+            normalized.contains("create branch") ||
+            normalized.contains("replace branch") ||
+            normalized.contains("create tag")))
```

Comment:
I think there is a fix for this at #6728 , might want to rebase after that is merged

Reply:
got it.